File: PB - More Exercises GO/1.0 Simple Operations and Calculations - More Exercises/05. Training Lab.go
package main

import "fmt"

func main() {
	var height, width float32
	fmt.Scan(&height, &width)
	hallway := 100
	lostWorkPosition := 3.0
	workPositionWidth := 70.0
	workPositionHeight := 120

	height *= 100
	width *= 100
	width -= float32(hallway)

	rest := int(width) % int(workPositionWidth)
	width -= float32(rest)
	bureauCount := width / float32(workPositionWidth)

	rest = int(height) % workPositionHeight
	height -= float32(rest)
	rowCount := height / float32(workPositionHeight)

	fmt.Println(bureauCount*rowCount - float32(lostWorkPosition))
}

/*
name   :05. Training Lab
input  :15 8.9
output :129
*/

File: PB - More Exercises Python/1.0 Simple Operations and Calculations - More Exercises/05. Training Lab.py
height = float(input()) * 100
width = float(input()) * 100 - 100
rest = width % 70
width -= rest
bureauCount = width / 70
rest = height % 120
height -= rest
rowCount = height / 120
print(bureauCount * rowCount - 3)

#---------------------------------------------(2)---------------------------------------
print(int(float(input()) * 100 / 120) * int((float(input()) * 100 - 100) / 70) - 3)

'''
15
8.4
---------
15
8.9
'''   #129



